frontend/src/routes/(unauthenticated)/register/+page.svelte

```diff
@@ -4,12 +4,16 @@
   import CreateUser from '$lib/components/Users/CreateUser.svelte';
   import { goto } from '$app/navigation';
   import { register } from '$lib/user';
+  import RegisterWithGoogleButton from '$lib/components/RegisterWithGoogleButton.svelte';
+  import { page } from '$app/stores';
 
   async function onSubmit(): Promise<void> {
     await goto('/home', { invalidateAll: true }); // invalidate so we get the user from the server
   }
 </script>
 
 <TitlePage title={$t('register.title')}>
+  <RegisterWithGoogleButton href={`/api/login/google?redirectTo=${$page.url.pathname}`}/>
```

Comment:
isn't the page here going to refer to this page? This redirectTo is where the user will be sent after login or registration, we don't want to send them to the register page after registration. That said I think this currently gets ignored and once registration is finished we just send them home instead of to wherever the redirectTo url was set.

Reply:
Tim fixed this before I could get to it; thanks, Tim.